src/webgpu/capability_info.ts

```diff
@@ -71,79 +71,82 @@ export const kBufferUsages = numericKeysOf<GPUBufferUsage>(kBufferUsageInfo);
+  'rgba16float':           [        true,        ,        ,          ,      true,          ,          ,               8],
   // 128-bit formats
-  'rgba32uint':           [        true,        ,        ,          ,      true,          ,          ,              16],
-  'rgba32sint':           [        true,        ,        ,          ,      true,          ,          ,              16],
-  'rgba32float':          [        true,        ,        ,          ,      true,          ,          ,              16],
+  'rgba32uint':            [        true,        ,        ,          ,      true,          ,          ,              16],
+  'rgba32sint':            [        true,        ,        ,          ,      true,          ,          ,              16],
+  'rgba32float':           [        true,        ,        ,          ,      true,          ,          ,              16],
 } as const);
 /* prettier-ignore */
-const kTexFmtInfoHeader = ['renderable', 'color', 'depth', 'stencil', 'storage', 'copySrc', 'copyDst', 'bytesPerBlock', 'blockWidth', 'blockHeight',              'extension'] as const;
+const kTexFmtInfoHeader =  ['renderable', 'color', 'depth', 'stencil', 'storage', 'copySrc', 'copyDst', 'bytesPerBlock', 'blockWidth', 'blockHeight',              'extension'] as const;
 export const kSizedDepthStencilFormatInfo = /* prettier-ignore */ makeTable(kTexFmtInfoHeader,
-                          [        true,   false,        ,          ,     false,          ,          ,                ,            1,             1,                         ] as const, {
-  'depth32float':         [        true,   false,    true,     false,          ,     false,     false,               4],
-  'stencil8':             [        true,        ,   false,      true,          ,     false,     false,               1],
+                           [        true,   false,        ,          ,     false,          ,          ,                ,            1,             1,                         ] as const, {
+  'depth32float':          [        true,   false,    true,     false,          ,     false,     false,               4,             ,              ,                         ],
+  'depth16unorm':          [        true,   false,    true,     false,          ,     false,     false,               2,             ,              ,                         ],
+  'stencil8':              [        true,        ,   false,      true,          ,     false,     false,               1,             ,              ,                         ],
+  'depth24unorm-stencil8': [        true,   false,    true,      true,          ,     false,     false,               4,             ,              ,  'depth24unorm-stencil8'],
+  'depth32float-stencil8': [        true,   false,    true,      true,          ,     false,     false,               8,             ,              ,  'depth32float-stencil8'],
```

Comment:
GLES spec says it is 8. But it might be useless because it is used by copy, but depth/stencil formats can not be used as copySrc and copyDst according to this table ( if I understand correctly).

Reply:
Your understanding is correct. So not sure if we should even include bytesPerBLock here. @kainino0x ?

Reply:
Hm, this is a weird situation. Really, bytesPerBlock only makes sense on a per-aspect basis. But these tables can't express that. I think(?) we can just consider `depth24unorm-stencil8` and `depth32float-stencil8` to be unsized formats at least for now.

Reply:
Done. and added your comment to explain why we put these two formats into unsized formats.